### PR TITLE
Updating one-line toast message

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] #1207
[Problem] - Old units
- unnecessary styles
- no media queries

[Solution] - Change units to @px_base
- Clean some styles
- Add media queries for one line toast messages

Signed-off-by: Kornelia Kobiela <korneliak.95@gmail.com>

### CODE REVIEW

File: src/css/profile/mobile/common/toast.less [hunk 1]
```diff
@@ -85,21 +71,6 @@
 	}
 }
 
-@-webkit-keyframes ui-smallpopup-show {
```
Comment (DohyungLim) [inline src/css/profile/mobile/common/toast.less:88]: Is it a keyframe that doesn't even need a webkit?

File: src/css/profile/mobile/common/toast.less [hunk 2]
```diff
@@ -2,20 +2,19 @@
 /* Popup Toast */
 .ui-popup.ui-popup-toast {
 	background-color: transparent;
+	margin-bottom: 0;
```
Comment (DohyungLim) [inline src/css/profile/mobile/common/toast.less]: Is the margin-bottom value 0 correct? Make sure that the spec comes out at the bottom and fits.

**Review by DohyungLim — COMMENTED** [review]
2 comment.

File: src/css/profile/mobile/common/toast.less (hunk 1 above)
Comment (korneliakobiela) [reply src/css/profile/mobile/common/toast.less:88]: As far as I know, this is added during build by autoprefixer.

File: src/css/profile/mobile/common/toast.less (hunk 2 above)
Comment (korneliakobiela) [reply src/css/profile/mobile/common/toast.less]: Margin bottom described in the spec is set in line 12.  We don't need additional margin.

File: src/css/profile/mobile/common/toast.less [hunk 3]
```diff
@@ -24,25 +24,22 @@
...
-
+			min-height: 44 * @px_base;
+			padding: 0 * @px_base 20 * @px_base;
+			background-color: var(--btn-toast-background);
 			.ui-btn {
-				padding: 4 * @px_base 20 * @px_base 4 * @px_base 0 * @px_base;
+				padding: 4 * @px_base 0 * @px_base 4 * @px_base 20 * @px_base;
+				margin: 0;
+				width: fit-content;
 				color: @color_toast_button_text;
```
Comment (TomaszLukawskiSam) [inline src/css/profile/mobile/common/toast.less]: Use css variable to define this color.

File: src/css/profile/mobile/common/toast.less [hunk 4]
```diff
@@ -57,15 +54,39 @@
...
-					left: -4 * @unit_base;
-					top: 50%;
-					transform: translate(0, -50%);
-				}
+			}
+		}
+	}
+}
+
+@media all and (max-width: 479px) {
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/toast.less]: How about moving the @media all and (max-width: 479px) { under the .ui-popup-content like as
```
.ui-popup.ui-popup-toast {
  .ui-popup-content { 
    @media all and (max-width: 479px) {
    // Changes
    }

    @media all and (min-width: 480px) and (max-width: 959px) {
    // Changes
    }
```

File: src/css/profile/mobile/base.less [hunk 5]
```diff
@@ -117,4 +119,5 @@
     --on-off-switch-on-disabled-track-background: @on-off-switch-on-disabled-track-background;
     --on-off-switch-off-track-background: @on-off-switch-off-track-background;
     --on-off-switch-off-disabled-track-border: @on-off-switch-off-disabled-track-border;
+
```
Comment (DohyungLim) [inline src/css/profile/mobile/base.less]: remove empty line.

**Review by DohyungLim — COMMENTED** [review]
1 comment.

**Review by DohyungLim — COMMENTED** [review]
and I think not found problem.

File: src/css/profile/mobile/common/toast.less (hunk 2 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/toast.less:5]: @korneliakobiela 
Are you setting margins elsewhere?
If not, it is better to delete margin-bottom.

Comment (DohyungLim) [reply src/css/profile/mobile/common/toast.less]: Are you setting margins elsewhere?
If not, it is better to delete margin-bottom.

File: src/css/profile/mobile/common/toast.less [hunk 6]
```diff
@@ -24,25 +24,22 @@
...
 			text-align: left;
-			min-width: 532 * @unit_base;
-			padding: 12 * @px_base 20 * @px_base;
-
+			min-height: 44 * @px_base;
+			padding: 0 * @px_base 20 * @px_base;
+			background-color: var(--btn-toast-background);
 			.ui-btn {
-				padding: 4 * @px_base 20 * @px_base 4 * @px_base 0 * @px_base;
+				padding: 4 * @px_base 0 * @px_base 4 * @px_base 20 * @px_base;
```
Comment (DohyungLim) [inline src/css/profile/mobile/common/toast.less]: Use only 0 instead of (0 * @px_base).

File: src/css/profile/mobile/common/toast.less (hunk 1 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/toast.less:88]: ok. I got it.

**Review by DohyungLim — COMMENTED** [review]
2 comment.

File: src/css/profile/mobile/common/toast.less (hunk 2 above)
Comment (korneliakobiela) [reply src/css/profile/mobile/common/toast.less]: We need it to remove margins from `ui-popup` class.

File: src/css/profile/mobile/common/toast.less (hunk 4 above)
Comment (korneliakobiela) [reply src/css/profile/mobile/common/toast.less]: Thank you for your comment. It was helpful

File: src/css/profile/mobile/common/toast.less (hunk 6 above)
Comment (korneliakobiela) [reply src/css/profile/mobile/common/toast.less]: Done

File: src/css/profile/mobile/base.less (hunk 5 above)
Comment (korneliakobiela) [reply src/css/profile/mobile/base.less]: Done!

File: src/css/profile/mobile/common/toast.less (hunk 3 above)
Comment (korneliakobiela) [reply src/css/profile/mobile/common/toast.less]: Done

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Pull request merged**